modules/common/react/lib/styles/focusRing.ts

```diff
@@ -99,9 +65,17 @@ interface ThemedFocusRingOptions {
  *
  * @returns {CSSObject} the css object for the focus ring style
  */
-export function themedFocusRing(
-  theme: CanvasTheme,
-  options: ThemedFocusRingOptions = {}
+export default function focusRing(
+  theme: CanvasTheme = useTheme(),
```

Comment:
theme was passed as first param because it is later used int he second param has a default value, I'm open to re working this

Reply:
Maybe let's change this to the second param. `focusRing({}, theme)` is a lot less common than `focusRing(undefined, {...})`, so I think this will be a lot cleaner.